src/web/components/MostViewed/MostViewedFooter/MostViewedFooterGrid.tsx

```diff
@@ -48,6 +49,12 @@ const selectedListTab = (pillar: CAPIPillar) => css`
 	transition: box-shadow 0.3s ease-in-out;
 `;
 
+// Used for the deeply read test
+const selectedDeeplyListTab = css`
```

Comment:
This is a take it or leave it but there's a vague convention to name these css variables with a `Styles` suffix, eg. `listStyles`. It can help as code scales to know what variables are what. For example, `selectedDeeplyListTab` _could_ refer to an index.

But again, this isn't a rule so whatever you feel best is fine with me

Reply:
Fair point, I'll change it to keep it up to the conventions we have.

Reply:
I've also updated the existing `selectedListTab` to `selectedListTabStyles`